app/test/unit/git/diff-test.ts

```diff
@@ -348,6 +348,7 @@ describe('git/diff', () => {
         repo.path
       )
 
+      // change config on-the-fly to trigger the line endings change warning
       await GitProcess.exec(['config', 'core.autocrlf', 'true'], repo.path)
```

Comment:
This is the only place I've left untouched by this PR, but I've added a better comment here to explain why it's necessary to do this mid-test rather than as part of the repository setupo.